pkg/alertmanager/alertmanager_metrics_test.go

```diff
@@ -100,40 +108,43 @@ func TestAlertmanagerMetricsStore(t *testing.T) {
 		cortex_alertmanager_nflog_snapshot_size_bytes 111
 		# HELP cortex_alertmanager_notification_latency_seconds The latency of notifications in seconds.
 		# TYPE cortex_alertmanager_notification_latency_seconds histogram
-		cortex_alertmanager_notification_latency_seconds_bucket{le="1"} 15
-		cortex_alertmanager_notification_latency_seconds_bucket{le="5"} 21
-		cortex_alertmanager_notification_latency_seconds_bucket{le="10"} 23
-		cortex_alertmanager_notification_latency_seconds_bucket{le="15"} 25
-		cortex_alertmanager_notification_latency_seconds_bucket{le="20"} 27
-		cortex_alertmanager_notification_latency_seconds_bucket{le="+Inf"} 27
-		cortex_alertmanager_notification_latency_seconds_sum 99.9
-		cortex_alertmanager_notification_latency_seconds_count 27
+		cortex_alertmanager_notification_latency_seconds_bucket{le="1"} 16
```

Comment:
These tests had drifted from the actual a bit, they also did not cover the `telegram` notifier. I fixed the drift in the tests while I was in the area. There will be a few extra changes in the expected metrics due to this.